pkg/config/config_local_test.go

```diff
@@ -277,4 +277,22 @@ var _ = Describe("Config Local", func() {
 		gomega.Expect(err).To(gomega.BeNil())
 		defer os.Remove(tmpfile)
 	})
+	It("Default Umask", func() {
+		// Given
+		// When
+		config, err := NewConfig("")
+		// Then
+		gomega.Expect(err).To(gomega.BeNil())
+		gomega.Expect(config.Containers.Umask).To(gomega.Equal("0022"))
+	})
+	It("should fail on bad Umask", func() {
```

Comment:
Can you add a umask to one of the config files and make sure that it is read correctly.

umask="0002"

Reply:
@rhatdan updated